src/test/java/neworangehrm/com/opensourcehrm/steps/DashboardSteps.java

```diff
@@ -21,7 +18,8 @@
 
     public DashboardSteps() {
         When("^I view welcome message$", () -> {
-            state.put("name", driverUtil.getDriverInstance().findElement(getElementLocator("welcomemessage")).getText());
+            state.put("name", driverUtil.getDriverInstance().findElement(PageUtil.commonWebPage.getElementLocator("welcomemessage")).getText());
+            //state.put("name", driverUtil.getDriverInstance().findElement(By.id("welcome")).getText());
         });
         Then("^Username is displayed$", () -> {
```

Comment:
How about making this step generic? like
Then("\"([^\"]*)\" is displayed$", (String someText) ->

Reply:
Accepted & Fixed.
Changed it to 
Then("^\"([^\"]*)\" text is displayed$", (String someText) -> {
Assert.assertTrue("Welcome message is missing user name", state.get("name").contains(someText));